functions.sh

```diff
@@ -3296,12 +3296,12 @@ report_statistic() {
   if [ "$1" -a "$2" -a "$3" -a "$4" -a "$5" ]; then
     REPORTSRV="$1"
 
-    STANDARDIMAGE="`ls -1 "$IMAGESPATH" |grep "$2"`"
+    STANDARDIMAGE="$(ls -1 "$IMAGESPATH" |grep "$2")"
```

Comment:
I know what positional parameters in bash are. My question was what kind of value we can expect for the parameter.
It's always hard to do such a cleanup if one doesn't know what the input can be and how the output should look like.
@bastelfreak can you create a new feature request giving every function an header about what input and output are? both in type and form. thank you